src/protocol/mod.rs

```diff
@@ -7,6 +7,7 @@
 //! to serialize the emssages it produces.
 pub mod echo_broadcast;
 pub mod errors;
+pub mod helpers;
```

Comment:
```suggestion
pub(crate) mod helpers;
```
no need to expose this outside the crate